DESCRIPTION

```diff
@@ -15,6 +15,7 @@ LinkingTo: Rcpp
 Imports:
     Rcpp,
     methods, 
-    Matrix
+    Matrix,
+    assertthat
```

Comment:
I would prefer not to have another dependency for argument checking. All those `assert_that` can also be modelled by `stopifnot`.  Would be great if you could revert that `asserthat` change.

What is better though, is to give a user better error messages instead of an expression that failed. E.g.
` if (!is.numeric(obj)) { stop("'obj' is not a numeric vector", call. = FALSE) }`
But that can also be a different PR.

Reply:
I think assertthat package is now widely used in every package Hardy W touches (which is a lot), so I would not hesitate much in using it as a dependancy.

it provides messages you want for free.

Reply:
I looked at the reverse dependencies and it seems that not that many of the tidyverse package use the package. But fine, let's pull in that dependency.